app/src/local-resources/runs/constants.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  RUN_STATUS_AWAITING_RECOVERY,
+  RUN_STATUS_AWAITING_RECOVERY_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_AWAITING_RECOVERY_PAUSED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_FAILED,
+  RUN_STATUS_FINISHING,
+  RUN_STATUS_IDLE,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_RUNNING,
+  RUN_STATUS_STOP_REQUESTED,
+  RUN_STATUS_STOPPED,
+  RUN_STATUS_SUCCEEDED,
+} from '@opentrons/api-client'
+
+import type { RunStatus } from '@opentrons/api-client'
+
+export const START_RUN_STATUSES: RunStatus[] = [
+  RUN_STATUS_IDLE,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+]
+export const RUN_NOT_STARTED_STATUSES: RunStatus[] = [
+  RUN_STATUS_IDLE,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+]
+export const NEEDS_CONFIRMATION_STATUSES: RunStatus[] = [
+  RUN_STATUS_IDLE,
+  RUN_STATUS_STOPPED,
+]
+export const RUN_AGAIN_STATUSES: RunStatus[] = [
+  RUN_STATUS_STOPPED,
+  RUN_STATUS_FINISHING,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_IDLE,
+  RUN_STATUS_FAILED,
+  RUN_STATUS_SUCCEEDED,
+]
+export const RECOVERY_STATUSES: RunStatus[] = [
+  RUN_STATUS_AWAITING_RECOVERY,
+  RUN_STATUS_AWAITING_RECOVERY_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_AWAITING_RECOVERY_PAUSED,
+]
+export const DISABLED_STATUSES: RunStatus[] = [
+  RUN_STATUS_FINISHING,
+  RUN_STATUS_STOP_REQUESTED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+  ...RECOVERY_STATUSES,
+]
+export const CANCELLABLE_STATUSES: RunStatus[] = [
+  RUN_STATUS_RUNNING,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_IDLE,
+  RUN_STATUS_AWAITING_RECOVERY,
+  RUN_STATUS_AWAITING_RECOVERY_PAUSED,
+  RUN_STATUS_AWAITING_RECOVERY_BLOCKED_BY_OPEN_DOOR,
+]
+export const TERMINAL_STATUSES: RunStatus[] = [
+  RUN_STATUS_STOPPED,
+  RUN_STATUS_SUCCEEDED,
+  RUN_STATUS_FAILED,
+]
+
+export const ACTIVE_PROTOCOL_STATUSES: RunStatus[] = [
```

Comment:
```suggestion
export const RUNNING_RECOVERY_STATUSES: RunStatus[] = [
```
Nit, just easier to understand what this group of statuses means at first glance.